src/main/java/lotto/domain/type/ResultType.java

```diff
@@ -27,13 +30,13 @@ public int getLottoMatchCount() {
         return lottoMatchCount;
     }
 
-    public static ResultType findType(int matchCount){
-        return Arrays.stream(values())
-                .filter(resultType -> resultType.hasCount(matchCount))
+    public static ResultType findType(int matchCount,boolean isBonusMatch){
+        return Arrays.stream(values()).
+                filter(resultType -> resultType.eqMachCount(matchCount) && resultType.isBonusMatch == isBonusMatch)
```

Comment:
```suggestion
                filter(this::findMatch)
```

-> 뒤 부분을 함수로 빼주면 더 깔끔할 것 같아요

Reply:
깔끔하게 짜는 연습을 더 많이 해야겠네요.. 좋은 지적 감사합니다!